xlators/cluster/afr/src/afr-common.c

```diff
@@ -1118,7 +1118,7 @@ __afr_inode_read_subvol_get(inode_t *inode, xlator_t *this, unsigned char *data,
     return ret;
 }
 
-int
+static int
 __afr_inode_split_brain_choice_get(inode_t *inode, xlator_t *this,
```

Comment:
nit: this one is called only once, from afr_inode_split_brain_choice_get() which by itself is static - can probably be folded into it? (I've done it here for example - https://github.com/gluster/glusterfs/pull/3327/files )

Reply:
@mykaul do you suggest incoporating your changes of altering the return type of __afr_inode_ctx_get() to resolve the above or simply clubbing the two funcs to resolve the above in this patch?

Reply:
The latter, if anything (or we can keep it for later).